test/sys/test_socket.rs

```diff
@@ -90,9 +90,9 @@ pub fn test_abstract_uds_addr() {
 pub fn test_getsockname() {
     use nix::sys::socket::{socket, AddressFamily, SockType, SockFlag};
     use nix::sys::socket::{bind, SockAddr};
-    use tempdir::TempDir;
+    use tempfile;
```

Comment:
This isn't necessary, correct?

Reply:
@Susurrus I don't understand what you mean?

Reply:
Sorry, don't know what I was talking about here. But since none of these are conditionally compiled I'd suggest we put a single `use` statement at the top of the file and get rid of all of the additional ones.